test/TestConfig/scripts/tools/getDependencies.pl

```diff
@@ -106,7 +106,7 @@
 	sha1 => 'bfcb96281ea3b59d626704f74bc6d625ff51cbce'
 );
 my %asmtools = (
-	url => 'https://ci.adoptopenjdk.net/view/Dependencies/job/asmtools/lastSuccessfulBuild/artifact/asmtools.jar',
+	url => 'https://ci.adoptopenjdk.net/view/Dependencies/job/asmtools/107/artifact/asmtools.jar',
```

Comment:
Instead of hardcode the build#, I think we should update the sha1 to match with the latest one. Jenkins build does not keep history forever.

Reply:
Agreed, see my comment at #2933